index.bs

```diff
@@ -1432,8 +1429,131 @@ following additional fields:
 Sessions {#streaming-sessions}
 ------------------------------------
 
-TODO
+To start a streaming session, a sender may send a
+streaming-session-start-request message with the following properties:
 
+: streaming-session-id
+:: Identifies the streaming session.  Must be unique for the (sender,
+    receiver) pair.  Can be used later to modify or terminate a
+    streaming session.
+
+: stats-interval
+:: Indicates the frequency the receiver should send stats messages to
+    the sender.
+
+: stream-offers
+:: Indicates the streams that the receiver can request from the sender.
+
+Each stream offer contains the following properties:
+
+: media-stream-id
+:: Identifies the media stream b	eing offered.  Must be unique within
+    the streaming session.  Can be used by the receiver to request the
+    media session.
+
+: friendly-name
+:: An optional name intended to be shown to a user, such that the
+    receiver may allow the user to choose which media streams to
+    receive, or if they are received automatically by the receiver,
+    give the user some information about what th e media stream is.
+
+: audio
+:: A list of audio encodings offered.  An audio encoding is a series
+    of encoded audio frames.  Encodings define properties needed by
+    the receiver to know how to decode the encoding, such as codec and
+    sample rate.  They can differ by codec and related properties,
+    but should be different encodings of the same audio.
+
+: video
+:: A list of video encodings offered.  A video encoding is a series of
+    encoded video frames.  Encodings define properties needed by the
+    receiver to know how to decode the encoding, such as codec and
+    default duration.  They can differ by codec and potentially other
+    properties, but should be different encodings of the same video.
+
+
+Each audio encoding offered defines the following properties:
+
+: encoding-id
+:: Identifies the audio encoding being offered.  Must be unique within
+    the media stream.  Can be the receiver to request an encoding.
+
+: codec
+:: The name of the codec used by the encoding.
+
+: time-scale
+:: The time scale used by all audio frames.  This allows senders to
+    make audio-frame messages smaller by not including the time scale
+    in each one.
+
+: default-duration:
+:: The duration of an audio frame .  This allows senders to make
+    audio-frame messagse smaller by not including the duration for
+    audio-frame messages that have the default duration.
+
+Each video encoding offered defines the following properties:
+
+: encoding-id
+:: Identifies the video encoding being offered.  Must be unique within
+    the media stream.  Can be the receiver to request an encoding.
```

Comment:
Similar comments as above for encoding-id

Reply:
Similar to above, as long as it's unique to the media stream, the receiver can demux based on it and the sender knows all the IDs the receiver knows because it picked them.

Reply:
I included that one as well.